Gemfile

```diff
@@ -24,7 +24,7 @@ gem "cheffish", ">= 17"
 group(:omnibus_package) do
   gem "appbundler"
   gem "rb-readline"
-  gem "inspec-core-bin", ">= 5" # need to provide the binaries for inspec
```

Comment:
@johnmccrae Why are we updating the pin for `inspec-core-bin`? That is another gem.
The one we want is `inspec-core` at 5.21.29 (which is already at it)